src/common/polkadot-provider/models/genetic-analisys/index.ts

```diff
@@ -0,0 +1,49 @@
+import { convertSubstrateNumberToNumber } from '../../';
+import { GeneticAnalysisStatus } from './genetic-analysis-status'
+
+export class GeneticAnalysis {
+  constructor(anyJson: any) {
+    this.genetic_analysis_tracking_id = anyJson.geneticAnalysisTrackingId;
+    this.genetic_analystId = anyJson.geneticAnalystId;
```

Comment:
use snake case for `id` in `genetic_analystId` and `genetic_analysis_orderId`

Reply:
also typo for the file name

Reply:
done